### PR TITLE
Fixes the bazel_apple_test

### PR DESCRIPTION
**Pull request by dmishe**
Instead of default SDK version, use currently installed one.

### CODE REVIEW

File: src/test/shell/bazel/bazel_apple_test.sh [hunk 1]
```diff
@@ -28,15 +28,18 @@ fi
 
 function set_up() {
   copy_examples
+  IOS_SDK_VERSION=$(xcrun --sdk iphoneos --show-sdk-version)
```
Comment (damienmg) [inline src/test/shell/bazel/bazel_apple_test.sh]: setup_objc_test_support would do that for you so we don't have that code in 2 places.

Comment (dmishe) [inline src/test/shell/bazel/bazel_apple_test.sh]: Done.

**Comment by damienmg** [comment]
LGTM

**Comment by damienmg** [comment]
This was merged and fixed the issue. Thanks again for the quick fix!

**Pull request closed without merging**